uint8 red
uint8 green
uint8 blue
uint8 blink_rate
---
bool success
